macros/cross_db_utils/ceil.sql

```diff
@@ -0,0 +1,11 @@
+{% macro ceil(field) -%}
+  {{ return(adapter.dispatch('ceil', packages = dbt_utils._get_utils_namespaces())(fields)) }}
```

Comment:
Ah, this argument as to match the arguments of the macro. This syntax is... not great 😬 

```suggestion
  {{ return(adapter.dispatch('ceil', packages = dbt_utils._get_utils_namespaces())(field)) }}
```